src/index.js

```diff
@@ -0,0 +1,172 @@
+// Accessing DOM/HTML Elements
+
+const upArrow=document.getElementById("up-arrow");
+const downArrow=document.getElementById("down-arrow");
+const temperature=document.getElementById("temp");
+const summer=document.getElementById("summer");
+const spring=document.getElementById("spring");
+const fall=document.getElementById("fall");
+const winter=document.getElementById("winter");
+const cityName=document.getElementById("city-name");
+const userSearch=document.getElementById("city-search");
+const callApiBtn=document.getElementById("real-temp-btn");
+const resetCityBtn=document.getElementById("reset-btn");
+const getWeatherBtn=document.getElementById("get-weather-btn");
+const skyDropDown=document.getElementById('sky-dropdown');
+const Sunny=document.getElementById('sunny');
+const Snowy=document.getElementById('snowy');
+const Rainy=document.getElementById('rainy');
+const Cloudy=document.getElementById('cloudy');
+const defaultSky=document.getElementById('default-sky');
+
+// ---------------------------
+
+let temp=0;
+colors();
+getApi();
+
+function getApi() {
+    let latitude;
+    let longitude;
+        axios
+        .get(`http://127.0.0.1:5000/location?q=${userSearch.value}`)
+        .then(function apiResponse(response) {
+            latitude = response.data[0].lat;
+            longitude = response.data[0].lon;
+
+            axios
+            .get(`http://127.0.0.1:5000/weather?&lat=${latitude}&lon=${longitude}`)
+            .then((res) => {
+                const fahrenheitConversion = Math.floor(
+                ((res.data.main.temp - 273.15) * 9) / 5 + 32);
+                temperature.textContent = fahrenheitConversion;
+                temp = fahrenheitConversion;
+                seasonChange()
+                })
+            })
+            .catch((error) => {
+                alert(error);
+            })    
+};
```

Comment:
Nice promise chaining for these functions, Tiffany! I have one refactoring advice for you and that is to implement the `async &c await` functionality of Javascript in order to make your code more concise and readable. doing something like: 

```Javascript
const getAPI = async () => {
  let lat, long;
  
  const response = await axios.get('http://127.0.0.1:5000/location', {
      params: {
        q: userSearch.value,
        format: 'json',
      }})
  
  if (response?.data) {
    lat = response.data[0].lat;
    long = response.data[0].lon;
    
    const res = await axios.get(`http://127.0.0.1:5000/weather?&lat=${latitude}&lon=${longitude}`)
    
    const tempConv = convert(res.data.main.temp)
    temp = tempConv
    seasonChange()
  } else {
    console.log('Error in find the latitude and longitude!');
  }
}

```
Using the `await` keyword lets us stop the global execution for the function it is in and wait for the promise to be fulfilled. This allows us to cut out the promise chaining here and just save return information from the axios calls as variables. We can then use said variables like we ordinarily would in the `then` chain of the promise. (the `?` in `response?.data` is just javascript syntax for checking if the key are trying to access is actually in the object)